src/collections/programs/lfx-2024/lfx-2024.mdx

```diff
@@ -233,6 +233,85 @@ Understand that your challenges will be promoted through CNCF channels, reviewed
 - LFX URL: https://mentorship.lfx.linuxfoundation.org/project/1a620529-f2be-4a6f-8b4d-0562731cb840
 
 
+## LFX Mentorship 2024 Fall Projects
+<br />
+
+### CNCF TAG Network
+<br/>
+
+#### Interrelating Kubernetes Resources: Identifying relationships between all standard and custom resources
+- Description: The OpenAPI specifications for Kubernetes provides taxonomy, but augmenting a graph data model with formalized ontologies enables any number of capabilities, one of the more straightforward is the inferencing requisite for natural language processing, and consequently, a human-centric query / response interaction becomes becomes possible. More importantly, more advanced systems can be built when a graph data model of connected systems is upgraded to be a knowledge semantic graph.
+- Expected Outcome: 
+  - YAML-formatted definition of one or more relationships per Kubernetes resource.
+  - Documentation of each relationship - per component.
+  - Development of new types of genealogies - new types of ways in which resources relate to one another and how these relationships might be visualized.
+  - Verification of functional relationships
+- Recommended Skills: DevOps, Kubernetes Administration, Light familiarity with all of the CNCF projects and a desire to study each project and their operators/resources.
+- Mentor(s):
+  - Mentor Name: Uzair Shaikh (@muzairs15, muzair.shaikh810@gmail.com), Lee Calcote (@leecalcote, leecalcote@gmail.com),
```

Comment:
A quick update in mentors section link to their profile in layer5

Reply:
Checkout prev internships section

Reply:
Member profile?